### PR TITLE
Clarify eTRNG usage

### PR DESCRIPTION
**Pull request by andreslagarcavilla**
Fixes https://github.com/chipsalliance/Caliptra/issues/152

### CODE REVIEW

**Review by FerralCoder — APPROVED** [review]

File: docs/CaliptraIntegrationSpecification.md [hunk 1]
```diff
@@ -471,6 +471,11 @@ See the Hardware specification for additional details.
 
 For SoCs that choose to not instantiate Caliptra’s embedded TRNG, we provide a TRNQ REQ HW API.
 
+**While the use of this API is convenient for early enablement, the current
+Caliptra hardware is unable to provide the same security guarantees with an
+external RNG. In particular, it is highly advisable to instantiate an internal
+RNG if ROM glitch protection is of importance.**
```
Comment (varuns-nvidia) [inline docs/CaliptraIntegrationSpecification.md:477]: should this also s/RNG/TRNG

Comment (andreslagarcavilla) [reply docs/CaliptraIntegrationSpecification.md]: Fixed now

**Review by varuns-nvidia — APPROVED** [review]

File: docs/CaliptraIntegrationSpecification.md [hunk 2]
```diff
@@ -471,6 +471,11 @@ See the Hardware specification for additional details.
 
 For SoCs that choose to not instantiate Caliptra’s embedded TRNG, we provide a TRNQ REQ HW API.
 
+**While the use of this API is convenient for early enablement, the current
+Caliptra hardware is unable to provide the same security guarantees with an
+external TRNG. In particular, it is highly advisable to instantiate an internal
+TRNG if ROM glitch protection is of importance.**
```
Comment (steph-morton) [inline docs/CaliptraIntegrationSpecification.md:477]: `is of importance`-->`is important`

**Review by steph-morton — DISMISSED** [review]

File: docs/CaliptraIntegrationSpecification.md (hunk 2 above)
Comment (andreslagarcavilla) [reply docs/CaliptraIntegrationSpecification.md:477]: Fixed now, thanks

**Review by steph-morton — APPROVED** [review]

**Review by FerralCoder — APPROVED** [review]

**Pull request merged**